index.html

```diff
@@ -29,187 +29,215 @@ <h1 class="portfolio">Portfolio</h1>
+                    <img src="images/works/4.png" alt="project picture" class="works-img">       
+                </div>
+
+                <div class="works-content">
+                    <h3 class="works-title">Multi-Post Stories</h3>
+                    <h4 class="works-headline">CANOPY <span class="bullet"> Back End dev </span> <span>2015</span></h4> 
+                    <p class="works-description">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.
+                    </p>
+                    <ul class="element-container">
+                        <li class="elements">
+                            html
+                        </li>
+                        <li class="elements">
+                            css
+                        </li>
+                        <li class="elements">
+                            javascript
+                        </li>
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </div>
 
-            <div class="skill-content">
-                <ul class="skill-list">
-                    <li class="skill-item">
-                        <i class="html fa-brands fa-html5 fa-2xl"></i>
-                        <p>HTML</p>
-                    </li>
-                    <li class="skill-item">
-                        <i class="css fa-brands fa-css3-alt fa-2xl"></i>
-                        <p>CSS</p>
-                    </li>
-                    <li class="skill-item">
-                        <i class="js fa-brands fa-js-square fa-2xl"></i>
-                        <p>Javascript</p>
-                    </li>
-                </ul>
             </div>
-            <div class="skill">
-                <h3>Framework</h3> 
-                <i class="fa-solid fa-angle-right"></i>
+    </section>
+    <section class="footer-container">
+        <div class="about-container">
+            <div class="about-des">
+                <h2 class="about-title">About Myself</h2>
+                <p class="headline-paragraph">
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="connect">
+                    <h3 >LET’S CONNECT</h3>
+                    <ul class="icons">
+                        <li class="hide fa-brands fa-twitter fa-xl"></li>
+                        <li class="hide fa-brands fa-linkedin-in fa-xl"></li>
+                        <li class="hide fa-brands fa-github fa-xl"></li>
+                        <li class="hide fa-solid fa-m fa-xl"></li>
+                        <li class="hide fa-solid fa-hand-peace fa-xl"></li>
+                    </ul>
```

Comment:
- [ OPTIONAL ] I notice that all your icons are not the link, you need to use an `<a>` tag in each `<li>` tag :upside_down_face: :upside_down_face:.
- All the icons are not here just for design. For example, if you use the `<a>` tag you can add your Twitter link so that if someone visits your portfolio, they can easily find your contact just by clicking on the icons :grin: :grin: